### ISSUE TITLE
Most dependencies should be peer dependencies

### ISSUE THREAD

**Issue by bjbrewster** [ratson/react-intl-redux#23]
There is a React runtime validation error when importing react-redux twice that is caused by react-intl-redux not using peerDependencies.

https://github.com/reactjs/react-redux/issues/534

As far as I can see, there is no good reason to not make all the `package.json` "dependencies" as "peerDependencies" except for the "warning" package.  It just means users will have more freedom in importing these packages themselves.

**Comment by bitpoetryx1** [comment]
I have also experienced the exact same problem.
For a temporary fix I did the following:

Removed «myProjectDir»\node_modules\react-intl-redux\node_modules\
Changed react-intl-redux\package.json 

```
...
 "dependencies": {
    "react-intl": "^2.2.2",
    "react-redux": "^5.0.1",
    "redux": "^3.6.0",
    "warning": "^3.0.0"
  }
...
```
to
```
...
"dependencies": {
    "warning": "^3.0.0"
  },
  "peerDependencies": {
    "react-intl": "2.x",
    "react-redux": "5.x",
    "redux": "3.x"
  }
...
```
finally, npm install

No more Warning messages and less 12KB in the final bundle. OFC this is only a silly temporary fix...

**Comment by ratson** [comment]
Fixed in v0.4.0
